run_9_global_nexus_lakes.py

```
@@ -0,0 +1,22 @@
+
+hydrography_dir = r"D:\geoglows_v3\geoglows_v3\hydrography"
+global_dir = r"D:\geoglows_v3\geoglows_v3\hydroprahy_global"
+
+
+if __name__ == '__main__':
+   # Look into subdirs of hydrography_dir
+    lakes = glob.glob(os.path.join(hydrography_dir, '*', 'lakes_*.gpkg'))
+    nexus = glob.glob(os.path.join(hydrography_dir, '*', 'nexus_*.gpkg'))
+
+    # Read all the files
+    lakes_gdf: gpd.GeoDataFrame = pd.concat([gpd.read_file(l) for l in lakes])
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)